lib/auth/permissions.go

```diff
@@ -554,7 +568,8 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 			role.String(),
 			services.RoleSpecV3{
 				Allow: services.RoleConditions{
-					Namespaces: []string{services.Wildcard},
+					Namespaces:       []string{services.Wildcard},
+					KubernetesLabels: services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
KubeService should not be able to write all other kube service records, that would be a security breach.

Reply:
That also makes sense. Just to be clear though, if that would be a security breach then it already exists today - all of these roles currently have the wildcard labels applied in `CheckAndSetDefaults`